src/AstroImages.jl

```diff
@@ -2,9 +2,9 @@ __precompile__()
 
 module AstroImages
 
-using FITSIO, FileIO, Images
+using FITSIO, FileIO, Images, Interact
```

Comment:
Also, I think Interact needs to be added to the dependencies and update CI for this to go green.